lib/danger/danger_core/plugins/dangerfile_danger_plugin.rb

```diff
@@ -27,7 +27,7 @@ module Danger
   #
   # @example Run a Dangerfile from inside a repo
   #
-  #          danger.import_dangerfile(gitlab: "ruby-grape/danger")
+  #          danger.import_dangerfile(gitlab_project_id: 1345)
```

Comment:
That's a regression, up until 6.0.6 `gitlab:` worked. From `6.0.7` onward, one has to workaround the issue by using the project id.

Related issue: https://github.com/danger/danger/issues/1130